protocol/v1/qbft/instance/commit.go

```diff
@@ -80,6 +82,11 @@ func (i *Instance) uponCommitMsg() pipelines.SignedMessagePipeline {
 			i.decidedMsg = agg
 			// mark instance commit
 			i.ProcessStageChange(qbft.RoundStateDecided)
+			messageID := message.ToMessageID(i.GetState().GetIdentifier())
+			metricsDurationStageCommit.
```

Comment:
lets extract to a separate func

Reply:
yeah cause this is not part of the really pipeline so need to be more clear and easy to read